app/models/ext_management_system.rb

```diff
@@ -1010,10 +1010,5 @@ def build_authentication_by_role(options)
     update_authentication(creds, options)
   end
 
-  def clear_association_cache
-    @storages = nil
-    super
-  end
-
```

Comment:
Always fun when you see a `git-svn-id`

Reply:
Yeah, 2010 is a nice one... It's been a while since I saw a svn revision less than 20000. 🤣

Reply:
I think in this case we might be able to drop this because storages used to be through hosts, but we've added `ems_id` to `storages` so this is a proper relation now

Reply:
Agreed, `@storages` was manually set in `storages` method via `hosts` so it made sense but with an real association, it shouldn't be needed anymore